frontend/app_flowy/packages/flowy_infra_ui/lib/style_widget/text_field.dart

```diff
@@ -19,6 +19,7 @@ class FlowyTextField extends StatefulWidget {
   final bool autoClearWhenDone;
   final bool submitOnLeave;
   final Duration? debounceDuration;
+  final String errorText;
```

Comment:
```suggestion
  final String? errorText;
```

I think this will be better (the default value for this field can also be removed), and then in the TextField, you can simply do `errorText: widget.errorText,`

Reply:
Tried it, didn't work. It seems that `_FieldNameTextFieldState` is passing empty text and also maybe related to that `TextField` will check on not being `null` for `_hasError`.

Reply:
Ahh I see, so there is a need for the check. Could you do `state.errorText.isEmpty ? null : state.errorText` in `_FieldNameTextFieldState`? The parameter format for the `TextField` and `FlowyTextField` should be similar